awx/main/utils/external_logging.py

```diff
@@ -22,6 +22,8 @@ def construct_rsyslog_conf_template(settings=settings):
         spool_directory = '/var/lib/awx'
 
     max_bytes = settings.MAX_EVENT_RES_DATA
+    if settings.LOG_AGGREGATOR_RSYSLOGD_DEBUG:
+        parts.append('$DebugLevel 2')
```

Comment:
@ryanpetrello For this to work, doesn't the `-d` flag need to be passed when supervisor starts rsyslogd?  Or you could alternatively set the `RSYSLOG_DEBUGLOG` environment variable.

Reply:
I don't _think_ so?  I was able to turn this new setting on/off and see higher log verbosity in my testing.

Reply:
Will do.  The other thing that comes to mind here is that we may want to be explicit about _where_ these debug logs are going to with `RSYSLOG_DEBUGLOG=”/path/to/debuglog/debug.log”`

Where were these logs getting written to by default?  might be good to include that in the tool tip for that setting.

Reply:
They're written to rsyslogd's stdout/err just like its other logs, and managed through supervisord (which I think is what makes sense).

Reply:
https://www.rsyslog.com/doc/v8-stable/troubleshooting/debug.html#enabling-debug-via-rsyslog-conf shows that the we can set the debug on the config file the way that is being done here

Reply:
Yep @elyezer that's where I got this idea from while reading the docs.

Reply:
I just checked this out locally and it does work as is.  Sorry for the confusion.